no_libpng16 it not appropriate for MacPorts, which provides libpng16.

--- configure.orig	2018-03-28 12:28:08.000000000 -0700
+++ configure	2019-02-20 15:39:46.000000000 -0700
@@ -13527,7 +13527,6 @@
 #define OSX 1
 _ACEOF
 
-		no_libpng16=yes # workaround until we can upgrade the libpng used by bockbuild
 		;;
 
 	*)
